test_support/src/lib.rs

```diff
@@ -0,0 +1,60 @@
+use libcnb_test::{
+    assert_contains, BuildConfig, BuildpackReference, ContainerConfig, TestContext, TestRunner,
+};
+use std::time::Duration;
+
+const PORT: u16 = 8080;
+
+pub enum Builder {
+    Heroku20,
+    Heroku22,
+}
+
+pub fn get_function_invoker_buildpacks() -> Vec<BuildpackReference> {
+    vec![
+        BuildpackReference::Other(String::from("heroku/nodejs-engine")),
+        BuildpackReference::Other(String::from("heroku/nodejs-npm")),
+        BuildpackReference::Crate,
```

Comment:
This function only seems relevant for the function invoker buildpack. Additionally,`BuildpackReference::Crate` might point to `heroku/nodejs-engine` when called from that buildpack. Maybe this function belongs only in the buildpack tests that needs it?

Reply:
yes, it is only relevant to the invoker buildpack which is why it is named `get_function_invoker_buildpacks`.  i was envisioning the `test_support` crate as a place to store reusable utility functions for all the buildpacks in the project so that the actual integration test files have minimal clutter.